app/models/base_facility.rb

```diff
@@ -153,6 +161,18 @@ def build_result_set(bbox_num, type, services)
       TYPES.map { |facility_type| get_facility_data(conditions, type, facility_type, services) }.flatten
     end
 
+    def build_result_set_from_ids(ids)
```

Comment:
Not in your PR, but I notice in find_facility_by_id (line 146), we are doing some sorting/upcasing of the hours section, doesn't seem like that's getting covered in either of the query options. Maybe add a TODO for that?